### DISCUSSION TITLE
Docs and Reasons for the JWT API changes

### DISCUSSION THREAD

**Discussion by HJFinch** [go-kit/kit#1136] [Q&A]
Hey there,
I wanted to understand what the reasons behind the change in the jwt package were and how this should be used now?
Is there any recommendations on how to work with this (real examples?) as this kinda happened under the hood and I was a little surprised by it, after stuff looked different (or was not working anymore).

Cheers,
Finch

**Comment by peterbourgon** [comment]
Changes since when? Here are a few tickets that caused API changes, with rationale: https://github.com/go-kit/kit/issues/562, https://github.com/go-kit/kit/issues/526.

**Comment by peterbourgon** [comment]
@HJFinch Does that help? Any outstanding questions?
